python/cudf/cudf/core/multiindex.py

```diff
@@ -944,6 +944,18 @@ def is_monotonic_decreasing(self):
     def argsort(self, ascending=True, **kwargs):
         return self._source_data.argsort(ascending=ascending, **kwargs)
 
+    def sort_values(self, return_indexer=False, ascending=True, key=None):
```

Comment:
Is the docstring here inherited from `Index.sort_values`?

Reply:
Yep